doc/classes/EditorSettings.xml

```diff
@@ -1249,6 +1369,14 @@
 		<member name="text_editor/theme/highlighting/word_highlighted_color" type="Color" setter="" getter="">
 			The script editor's color for words highlighted by selecting them. Only visible if [member text_editor/appearance/caret/highlight_all_occurrences] is [code]true[/code].
 		</member>
+		<member name="text_editor/theme/line_spacing" type="int" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="text_editor/theme/line_spacing" type="int" setter="" getter="">3
			Vertical line separation used in text editors.
		</member>
```
Seems like it affects all text editors, even though I only see one usage in shader source visualiser.